### ISSUE TITLE
I have a problem for login in v7.6.1

### ISSUE THREAD

**Issue by van2727** [DIGITALCRIMINAL/UltimaScraper#1789]
I put xb-c in auth.json as well 

Auth (V1) Attempt 1/10
Access denied.
Auth (V1) Attempt 2/10
Access denied.
Auth (V1) Attempt 3/10
Access denied.
Auth (V1) Attempt 4/10
Access denied.
Auth (V1) Attempt 5/10
Access denied.
Auth (V1) Attempt 6/10
Access denied.
Auth (V1) Attempt 7/10
Access denied.
Auth (V1) Attempt 8/10
Access denied.
Auth (V1) Attempt 9/10
Access denied.
Auth (V1) Attempt 10/10
Access denied.
Opening Browser
Traceback (most recent call last):
  File "G:\win\OnlyFans-7.6.1\OnlyFans-7.6.1\start_ofd.py", line 104, in <module>
    asyncio.run(main())
  File "C:\Users\user\AppData\Local\Programs\Python\Python310\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "C:\Users\user\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 641, in run_until_complete
    return future.result()
  File "G:\win\OnlyFans-7.6.1\OnlyFans-7.6.1\start_ofd.py", line 86, in main
    api = await main_datascraper.start_datascraper(
  File "G:\win\OnlyFans-7.6.1\OnlyFans-7.6.1\datascraper\main_datascraper.py", line 67, in start_datascraper
    setup, subscriptions = await module.account_setup(
  File "G:\win\OnlyFans-7.6.1\OnlyFans-7.6.1\modules\onlyfans.py", line 117, in account_setup
    oflogin.login(auth, domain, auth.session_manager.get_proxy())
  File "G:\win\OnlyFans-7.6.1\OnlyFans-7.6.1\extras\OFLogin\start_ofl.py", line 77, in login
    web_browser = launch_browser(
  File "G:\win\OnlyFans-7.6.1\OnlyFans-7.6.1\extras\OFLogin\start_ofl.py", line 33, in launch_browser
    driver = webdriver.Firefox(firefox_profile=profile, executable_path=driver_path,
  File "C:\Users\user\AppData\Roaming\Python\Python310\site-packages\seleniumwire\webdriver\browser.py", line 45, in __init__
    super().__init__(*args, **kwargs)
  File "C:\Users\user\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\firefox\webdriver.py", line 174, in __init__
    self.service.start()
  File "C:\Users\user\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\common\service.py", line 98, in start
    self.assert_process_still_running()
  File "C:\Users\user\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\common\service.py", line 110, in assert_process_still_running
    raise WebDriverException(
selenium.common.exceptions.WebDriverException: Message: Service G:\win\OnlyFans-7.6.1\OnlyFans-7.6.1\extras\OFLogin\geckodriver.exe unexpectedly exited. Status code was: 64

**Comment by fknMega** [comment]
I get this too, someone has a solution?

**Comment by jerryhard** [comment]
same problem, after this OF unlog me and change SESS

**Comment by MrRawrrino89** [comment]
Yeah same problem mine just keeps running through the entire denied thing 
![image](https://user-images.githubusercontent.com/86880907/144756917-ad0f8a33-98b7-4a59-bd82-a3dbc5811da9.png)

**Comment by SirPenetrat0r** [comment]
I dunno, but have you tried clearing the cache etc. and then redownloading the gecko driver/folder an replacing the existing one in the OFLogin folder? 

I guess you already have the Geckodriver folder in there, correct? 

Also, if you have 2FA activated, try again with 2FA disabled.

**Comment by jerryhard** [comment]
UFF ... this is old vertsion ... i migrate today to new one ... need add X_BC parametr to auth login ...
maybe it wor fort his vertsion too ... but i migrate to python3.10 - only in OnlyFans/vars/ ... Linux ;-)) 

New version work but need X_BC in aut file ... maybe this work on old version too ... i am in new version ;-))

**Comment by van2727** [comment]
> I dunno, but have you tried clearing the cache etc. and then redownloading the gecko driver/folder an replacing the existing one in the OFLogin folder?
> 
> I guess you already have the Geckodriver folder in there, correct?
> 
> **---** I just checked my recent folder and I don't even have the geckodriver added like before - so maybe try with removing geckodriver files, and later maybe without 2FA
> 
> Also, if you have 2FA activated, try again with 2FA disabled.

I try with a clean version of my SO , and the error still happens :(

**Comment by anonimizer783** [comment]
This is failing on hcaptcha

**Comment by octrillian** [comment]
I found there is now a `bcTokenSha` key in the browser's localStorage.  If I delete that key in the browser, I am immediately logged out.  You would need a headless browser to replicate the localStorage API 😦

**Comment by vonhapless** [comment]
I was facing the same issue. My steps to fix it:

- Clone the new version from [https://github.com/DIGITALCRIMINALS/OnlyFans/](https://github.com/DIGITALCRIMINALS/OnlyFans/) (the new @DIGITALCRIMINAL repo). He explained [here](https://github.com/DIGITALCRIMINAL/OnlyFans/issues/1596#issuecomment-903489056) about his main account.
- Renamed `config.json` and `auth.json` to something different, like `config.json.old` and `auth.json.old`
- When running the script, you will see:
```
The .settings\config.json file has been updated. Fill in whatever you need to fill in and then press enter when done.

/usr/src/app/.profiles/OnlyFans/default/auth.json has been created. Fill in the relevant details and then press enter to continue.
```
- Updated the new `auth.json` with the `cookies`, `user_agent` and `x_bc` as normal. Attention for the fact of the property is `x_bc` and not `x-bc`
- Happy days!

**Comment by aroundtheblockchainer** [comment]
7.6.1 still requires loading and reloading of xbc and sess id every time you open a new instance of the ripper. This is a bit of a pain, but I'm guessing this is the only way round this problem for the time being.

**Comment by blanuk** [comment]
@vonhapless how get i the x_bc code? found nothing :/

**Comment by aroundtheblockchainer** [comment]
> @vonhapless how get i the x_bc code? found nothing :/

same way you get all the other info...CTR+SHIFT+J on Chrome>Network>scroll all the way down to the bottom of the page and you'll see xbc right there..

**Comment by blanuk** [comment]
found it, thanks. was on preview and not on header :/
Thanks a lot for fast reply and help :)

**Comment by LucioSaldivar** [comment]
> 7.6.1 still requires loading and reloading of xbc and sess id every time you open a new instance of the ripper. This is a bit of a pain, but I'm guessing this is the only way round this problem for the time being.

would you mind letting me know where and how to put xbc code? I have tried inside the "cookie" variable as well as tried assigning "x_bc" in the array and none worked.

**Comment by SirPenetrat0r** [comment]
> @aroundtheblockchainer
> 
> > 7.6.1 still requires loading and reloading of xbc and sess id every time you open a new instance of the ripper. This is a bit of a pain, but I'm guessing this is the only way round this problem for the time being.
> 
> would you mind letting me know where and how to put xbc code? I have tried inside the "cookie" variable as well as tried assigning "x_bc" in the array and none worked.

You need to download version 7.6.1 from here: 
https://github.com/DIGITALCRIMINALS/OnlyFans/releases

Just follow this "guide" I made: 
https://pdfhost.io/v/PXwOAw3Tp_Download_the_OnlyFans_Scraper

**Comment by LucioSaldivar** [comment]
> You need to download version 7.6.1 from here: https://github.com/DIGITALCRIMINALS/OnlyFans/releases
> 
> Just follow this "guide" I made: https://pdfhost.io/v/PXwOAw3Tp_Download_the_OnlyFans_Scraper

Scraper is not even installing

**Comment by SirPenetrat0r** [comment]
> > You need to download version 7.6.1 from here: https://github.com/DIGITALCRIMINALS/OnlyFans/releases
> > Just follow this "guide" I made: https://pdfhost.io/v/PXwOAw3Tp_Download_the_OnlyFans_Scraper
> 
> Scraper is not even installing

Well, your lack of provided information makes it quite hard to help in any way.

**Comment by LP560** [comment]
> > @aroundtheblockchainer
> > > 7.6.1 still requires loading and reloading of xbc and sess id every time you open a new instance of the ripper. This is a bit of a pain, but I'm guessing this is the only way round this problem for the time being.
> > 
> > 
> > would you mind letting me know where and how to put xbc code? I have tried inside the "cookie" variable as well as tried assigning "x_bc" in the array and none worked.
> 
> You need to download version 7.6.1 from here: https://github.com/DIGITALCRIMINALS/OnlyFans/releases
> 
> Just follow this "guide" I made: https://pdfhost.io/v/PXwOAw3Tp_Download_the_OnlyFans_Scraper

That fixed all the errors i was having in #1797

**Comment by jaysomm** [comment]
you need to look at the xhr's for login specifically to get the x_bc value. I'm curious if anyone knows that this x_bc var is?
